cmd/flag/verbose_flags.go

```diff
@@ -0,0 +1,36 @@
+// Copyright © 2018 Infostellar, Inc.
```

Comment:
`2019` (I haven't been looking in default in other PRs, maybe we should review all the files in another PR)

Reply:
I took care of the year, but missed to update it for this file. Other files should be OK.